pom.xml

```diff
@@ -19,7 +19,7 @@
         <java.level>8</java.level>
         <jackson.version>2.13.4</jackson.version>
         <!-- When updating the Jenkins version, also update io.jenkins.tools.bom dependency management and the README -->
-        <jenkins.version>2.319</jenkins.version>
+        <jenkins.version>2.321</jenkins.version>
```

Comment:
I think we should stay on `2.319` as that is an LTS. I'm not very keen to move to a weekly release for us.